aep/general/0155/aep.yaml

```diff
@@ -0,0 +1,9 @@
+---
+id: 155
+state: approved
+slug: idempotency
+created: 2019-05-06
+placement:
+  category: design-patterns
```

Comment:
I kind of feel like this could be in a more specific category, like "requests", since it's pertains to API requests. Design patterns feels a bit broader to me, like combining behaviors to create things like singletons.

Reply:
Sure thing, done.